newrelic-integration-e2e/main.go

```diff
@@ -15,15 +15,17 @@ const (
 	flagVerboseMode = "verbose_mode"
 	flagLicenseKey  = "license_key"
 	flagAgentDir    = "agent_dir"
+	flagRootDir     = "root_dir"
 )
 
 //go:embed resources/docker-compose.yml.tmpl
 var dockerComposeTemplate string
 
-func processCliArgs() (string, string, string, logrus.Level) {
+func processCliArgs() (string, string, string, string, logrus.Level) {
 	specsPathPtr := flag.String(flagSpecPath, "", "Relative path to the spec file")
 	licenseKeyPtr := flag.String(flagLicenseKey, "", "New Relic License Key")
 	agentDir := flag.String(flagAgentDir, "", "Directory used to deploy the agent")
```

Comment:
I personally prefer viper with a struct defined, but if this is simpler I am fine with it :)